src/rars/venus/MessagesPane.java

```diff
@@ -60,6 +61,7 @@ public class MessagesPane extends JPanel {
     private final JRadioButton buttonBatch;
     private final JButton inputTabClearButton;
     private final Box runioContent;
+    private final JTextArea programArguments;
```

Comment:
Are multi-line program arguments something we want to support? We could replace the JTextArea with a JTextField to "limit" it to a single line. Unsure if it's better. Thoughts?

INFO: Seems the parser just takes any whitespace with StringTokenizer. A new line would be considered a different argument.

Reply:
JTextField had issue with horizontal width and needs a fixed size, I did not manage to have a full width component so I used a JTextArea. JTextArea also look more integrated with the other tabs

I used `programArguments.getDocument().putProperty("filterNewlines", Boolean.TRUE);` that transform newline on input as space.

An alternative could be to have an argument per line and accept space as a normal character (we just split with newline).